src/components/Comments/hooks/useCommentAction.js

```diff
@@ -40,14 +44,13 @@ const useCommentAction = () => {
         }
       }
     } catch (e) {
-      if (__DEV__) {
-        console.log(e, 'eman');
-      }
+      console.log(e, 'eman');
     }
   };
 
   return {
-    showAlertDelete
+    showAlertDelete,
+    onDeleteCommentClicked
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a change in error handling. Previously, the console log was only executed in development mode. Now it will be executed in all environments which could potentially expose sensitive information. Consider reverting this change.
```diff
-      console.log(e, 'eman');
+      if (__DEV__) {
+        console.log(e, 'eman');
+      }
```
Also, the return statement of the hook has been modified to include `onDeleteCommentClicked`. Make sure this function is used appropriately wherever this hook is invoked.


<!-- This is an auto-generated comment by OSS CodeRabbit -->